Extractor/JsExtractor.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Bazinga\Bundle\JsTranslationBundle\Extractor;
+
+final class JsExtractor extends Extractor
+{
+    protected $sequence = '\\.trans[Choice]*\\(';
```

Comment:
Sure: Compare your regex:
https://regex101.com/r/9Ng6xR/1
with mine
https://regex101.com/r/wrJ9WE/1

Your regular expression does also match trans... where ... is any combination of the letters `C`, `h`, `o`, `i`, `c`, `e`. Mine looks specifically for `Choice`, in this order.